aspnetcore/tutorials/getting-started-with-NSwag.md

```diff
@@ -274,10 +274,12 @@ NSwag uses [Reflection](/dotnet/csharp/programming-guide/concepts/reflection), a
 
 [!code-csharp[](../tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.NSwag/Controllers/TodoController.cs?name=snippet_CreateAction)]
 
-The preceding action returns `ActionResult<T>`, but inside the action it's returning either [CreatedAtRoute](/dotnet/api/system.web.http.apicontroller.createdatroute). Since the controller is decorated with the [[ApiController]](/dotnet/api/microsoft.aspnetcore.mvc.apicontrollerattribute) attribute, a [BadRequest](/dotnet/api/system.web.http.apicontroller.badrequest) response is possible too. See [Automatic HTTP 400 responses](xref:web-api/index#automatic-http-400-responses) for more info. Data annotations are used to tell clients which HTTP status codes this action is known to return. Decorate the action with the following attributes:
+The preceding action returns `ActionResult<T>`, but inside the action it's returning [CreatedAtRoute](/dotnet/api/system.web.http.apicontroller.createdatroute). Since the controller is decorated with the [[ApiController]](/dotnet/api/microsoft.aspnetcore.mvc.apicontrollerattribute) attribute, a [BadRequest](/dotnet/api/system.web.http.apicontroller.badrequest) response is possible too. See [Automatic HTTP 400 responses](xref:web-api/index#automatic-http-400-responses) for more info. Data annotations are used to tell clients which HTTP status codes this action is known to return. Decorate the action with the following attributes:
```

Comment:
"inside" a prep in that position, so ...

> ... returns `ActionResult<T>`; but inside the action, it's ...

... or break the sentence there (+comma) ...

> ... returns `ActionResult<T>`. Inside the action, it's ...

Comma ...

> is possible, too.

Are we sticking with the format? ...

> For more information, see [Automatic HTTP 400 responses](xref:web-api/index#automatic-http-400-responses).

... and xref links.

The following suggestion uses the sentence break approach:

```suggestion
The preceding action returns `ActionResult<T>`. Inside the action, it's returning [CreatedAtRoute](xref:System.Web.Http.ApiController.CreatedAtRoute*). Since the controller is decorated with the [[ApiController]](xref:Microsoft.AspNetCore.Mvc.ApiControllerAttribute) attribute, a [BadRequest](xref:System.Web.Http.ApiController.BadRequest*) response is possible, too. For more information, see [Automatic HTTP 400 responses](xref:web-api/index#automatic-http-400-responses). Data annotations are used to tell clients which HTTP status codes this action is known to return. Decorate the action with the following attributes:
```